website/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' https://www.googletagmanager.com/ https://www.google-analytics.com/ https://static.hotjar.com/ https://script.hotjar.com/ http://adpxl.co/SyscD5JI/ https://slack.k8s.io/ http://s7.addthis.com/ https://z.moatads.com/ https://v1.addthisedge.com/ https://m.addthis.com/ wss://slack.k8s.io/ https://d33wubrfki0l68.cloudfront.net/ https://netlify-cdp-loader.netlify.app 'unsafe-eval' 'unsafe-inline'; style-src 'self' https://d33wubrfki0l68.cloudfront.net/css/a6538ee71bcce2d3f2480d5c4739d3aecc4f3118/ https://d33wubrfki0l68.cloudfront.net/ 'unsafe-inline'; object-src 'none'">
```

Comment:
https://d33wubrfki0l68.cloudfront.net/css/a6538ee71bcce2d3f2480d5c4739d3aecc4f3118/  => change it into *.cloudfront.net/
we can trust netlify i believe,

Reply:
Only the following subdomain is used`d33wubrfki0l68`. Why do we need to allow other subdomains?